x-pack/test/functional/apps/painless_lab/painless_lab_flyout.ts

```diff
@@ -21,9 +37,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('click show API request button and flyout should appear in page', async () => {
+      // replace the default script with a simpler one
+      await monacoEditor.setCodeEditorValue(TEST_SCRIPT);
       await testSubjects.click('btnViewRequest');
-
       await testSubjects.existOrFail('painlessLabRequestFlyoutHeader', { timeout: 10 * 1000 });
     });
+
+    it('validate request body is the expected', async () => {
+      const requestText = await testSubjects.getVisibleText('painlessLabFlyoutRequest');
+      expect(requestText).equal(TEST_SCRIPT_REQUEST);
+    });
+
+    it('validate response body is the expected', async () => {
+      await testSubjects.findService.clickByCssSelector('#response');
```

Comment:
```suggestion
      await find.clickByCssSelector('#response');
```

Wow, I did not even realize the test subjects svc exposed the find svc.
But, I'd prefer to see this written as I've suggested.
Of course, you'll need a `const find = getService('find');` at the top for this.